tutorials/auto_inertia_calculation.md

```diff
@@ -0,0 +1,163 @@
+the menu. Once you play the simulation it should look this:
+
+![Pendulum](files/auto_inertia/auto_inertia_pendulum.gif)
+
+This example world has two structurally indentical models. The pendulum link of both
+the models contain 3 cylindrical collision geometries:
+ - One on the top which forms the joint (pivot)
+ - A longer cylinder in middle
+ - One at the end which forms the bob of
+cylinder.
+Even, though they are indentical, the center of mass for both are different
+as they use different density values for the different cylinder collisions. One one
+hand, the upper joint collision of the pendulum on the left has the highest density
+which causes the center of mass to shift closer to the axis. While on the other hand,
+the bob collision of the pendulum on the right has the highest density which causes
+the center of mass to shift towards the end of the pendulum.
+This difference in mass distribution about the axis of rotation results in difference
+in the moment of inertia of the 2 setups and hence different angular velocities.
+
+## Mesh Inertia Calculation with Rolling Shapes Demo
+
+Let's try another example world, `auto_inertia_rolling_shapes.sdf`. This can be
+launched with `gz sim` using the following command:
+
+```bash
+gz sim auto_inertia_rolling_shapes.sdf
+```
+
+Once you launch and play the simulation, it should look something like this:
+
+![Rolling](files/auto_inertia/rolling_inertia_demo.gif)
+
+Here the right most shapes is a hollow cylinder (yellow). This model is loaded from
+[Gazebo Fuel](https://app.gazebosim.org/jasmeetsingh/fuel/models/Hollow%20Cylinder)
+and is made using a collada mesh of a hollow cylinder. Apart from this, we can
+see there is a solid cylinder, a solid sphere and a solid capsule. All of these are
+made using the `<geometry>` tag and have automatic inertia calculations enabled.
+Here, the moments of inertia for the hollow cylinder (which is convex mesh shape) is
+calculated and from the simulation we can see that it reaches the bottom of the
+incline last. This is physically accurate as the mass distribution for the hollow
+cylinder is concentrated at a distance from the axis of rotation (which passes through
+the center of mass in this case).
+
+## Key Points on Mesh Inertia Calculator
+
+Here are some key points to consider when using automatic inertia calculation with 3D Meshes:
+ * Water-tight triangle meshes are required for the Mesh Inertia Calculator.
+ * Currently, the mesh inertia is calculated about the mesh origin. Since the link
+ inertia value needs to be about the Center of Mass, the mesh origin needs to be set
+ at the Center of Mass (Centroid).
+ * Since the vertex data is used for inertia calculations, high vertex count would be
+ needed for near ideal values. However, it is recommended to use basic shapes with the
+ geometry tag (Box, Capsule, Cylinder, Ellipsoid and Sphere) as collision geometries to
+ reduce the load of calculations.

```

Comment:
When you say the load of calculations, it makes it sound like using high vertex count meshes only affects the performance of the auto MOI calcuclator, when in fact it will affect the overall simulation performance or RTF.

Reply:
Updated the doc in [6217798](https://github.com/gazebosim/gz-sim/pull/2119/commits/6217798a922afdfbbd7c690803b1774c041502b7)